cmd/pg-schema-diff/plan_cmd.go

```diff
@@ -29,6 +31,31 @@ const (
 	lockTimeoutInsertStatementKey      = "lock_timeout"
 )
 
+type outputFormat string
+
+const (
+	outputFormatPretty outputFormat = "pretty"
+	outputFormatJson   outputFormat = "json"
+)
+
+func (e *outputFormat) String() string {
+	return string(*e)
+}
+
+func (e *outputFormat) Set(v string) error {
+	switch v {
+	case "pretty", "json":
+		*e = outputFormat(v)
+		return nil
+	default:
+		return errors.New(`must be one of "pretty" or "json"`)
```

Comment:
nit: include the unknown value the user used. Also let's group with the other plan flag structs

Reply:
It looks like Cobra prepends this. I added the value the user used and it came out like this:
```
Error: invalid argument "test" for "--output-format" flag: must be one of "pretty" or "json", value passed in was "test"
```

I'm going to leave it out, but if you think it's useful because the error might be used outside of Cobra let me know and I can add it back in.